site/class-1-play/maroeska.svg

```diff
@@ -0,0 +1,125 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="500" height="500" viewBox="0 0 500 500">
+  <!--Update the title with your username-->
+  <title>@maroeska</title>
+
+  <!--Add stuff here-->
+    <circle
+        cx="240"
+        cy="240" 
+        r="100" 
+        fill="greenyellow" />
+    <rect
+        x="15" 
+        y="15" 
+        width="100" 
+        height="100" /> 
+    <line
+        x1="30"
+        y1="100"
+        x2="150"
+        y2="20"
+        stroke-width="10" 
+        stroke="red" /> 
+    <polygon
+        points="100,10 40,198 190,78 10,78 160,198"
+        fill="lime"
+        opacity="0.7" />
+    <path 
+        d="M 20 50 L 120 200 L 70 100 z" 
+        fill="aquamarine" />  
+    <text
+        x="0"
+        y="200"
+        font-family="arial" 
+        font-size="40" 
+        >bepbeppp</text>  
+    <g fill="yellow" stroke="black" stroke-width="10">
+        <circle cx="300" cy="300" r="70" />
+        <circle cx="400" cy="300" r="70" />    
+    </g>
+             
+  <!-- NOTHING TO SEE HERE... MOVE ALONG!!! -->
```

Comment:
Great, but could you remove this whole section too? So from the comment above (line 41) to `  // ]]></script>` (line 124).

Don’t remove the `</svg>` though!